### PR TITLE
feat: add fastfetch and include in comps; update hyfetch

### PR DESCRIPTION
**Pull request by madonuko**
I was about to obsolete `hyfetch` but I kinda realized we never updated it… so maybe we should try out this newer version

I also included `fastfetch` in comps so desktop users don't have to install it… I mean it's kinda bloaty but it's just 2.5 MiB… not sure

### CODE REVIEW

**Comment by lleyton** [comment]
you probably want to pull um41 into your branch and resolve conflicts, I'm pretty sure I already added fastfetch into one of my PRs

File: ultramarine/hyfetch/hyfetch.spec [hunk 1]
```diff
@@ -55,8 +57,11 @@ Recommends:     xwininfo
 %description -n hyfetch-neofetch
 %{summary}.
 %files -n hyfetch-neofetch
+%{_bindir}/neowofetch
 %{_bindir}/neofetch
 %{_mandir}/man1/neofetch.1.gz
+%{_mandir}/man1/neowofetch.1.gz
+/usr/lib/python*/site-packages/HyFetch-%version-py*.egg/
```
Comment (lleyton) [inline ultramarine/hyfetch/hyfetch.spec]: macro?

File: comps.xml [hunk 2]
```diff
@@ -54,6 +54,7 @@
       <packagereq type="default">sipa-fonts</packagereq>
       <packagereq type="default">ibus-mozc</packagereq>
       <packagereq type="default">mozc</packagereq>
+      <packagereq type="default">fastfetch</packagereq>
```
Comment (lleyton) [inline comps.xml]: already in ultramarine-product-common?

**Review by lleyton — CHANGES_REQUESTED** [review]

**Comment by lleyton** [comment]
Please fix the failing build

**Comment by madonuko** [comment]
I think gha is broken

**Comment by lleyton** [comment]
uh, wtf...

**Review by lleyton — APPROVED** [review]

**Comment by lleyton** [comment]
merged it because thought it might get GHA to actually work, seems like build still failing, can you look at it?

**Pull request merged**